CHANGELOG.md

```diff
@@ -3,6 +3,40 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.23](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.22...v4.0.0-alpha.23)
+
+_Mar 22, 2021_
+
+Big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 🇵🇱 Added plPL locale (#1117) @LarsKumbier
+- 🎁 Add onScrollBottom to support infinite loading (#1199) @DanailH
```

Comment:
This seems to be the biggest improvement of the release, we might want to mention it once, add a link to the documentation/record a video to demonstrate what it does.

Reply:
Updated. Added additional explanation and a link to the docs.